docs/en/migration.md

```diff
@@ -0,0 +1,572 @@
+
+```python
+visualizer = dict(
+    type='ClsVisualizer',
+    vis_backends=[
+        dict(type='LocalVisBackend'),
+        # Uncomment the below line to save the log and visualization results to TensorBoard.
+        # dict(type='TensorboardVisBackend')
+    ]
+)
+```
+
+New field **`default_scope`**: The start point to search module for all registries. The `default_scope` in MMClassification is `mmcls`. See [the registry tutorial](TODO) for more details.
+
+## Packages
+
+### `mmcls.apis`
+
+The documentation can be found [here](mmcls.apis).
+
+|       Function       |                     Changes                     |
+| :------------------: | :---------------------------------------------: |
+|     `init_model`     |                   No changes                    |
+|  `inference_model`   |                   No changes                    |
+|    `train_model`     |      Removed, use `runner.train` to train.      |
+|   `multi_gpu_test`   |       Removed, use `runner.test` to test.       |
+|  `single_gpu_test`   |       Removed, use `runner.test` to test.       |
+| `show_result_pyplot` |              Waiting for support.               |
+|  `set_random_seed`   | Removed, use `mmengine.runner.set_random_seed`. |
+|  `init_random_seed`  | Removed, use `mmengine.dist.sync_random_seed`.  |
+
+### `mmcls.core`
+
+The `mmcls.core` package is renamed to [`mmcls.engine`](mmcls.engine).
+
+| Sub package  |                                               Changes                                               |
+| :----------: | :-------------------------------------------------------------------------------------------------: |
+| `evaluation` |                           Removed, use the metrics in `mmcls.evaluation`.                           |
+|    `hook`    |                                    Moved to `mmcls.engine.hooks`                                    |
+| `optimizers` |                                 Moved to `mmcls.engine.optimizers`                                  |
+|   `utils`    | Removed, the distributed environment related functions can be found in the `mmengine.dist` package. |
+
+### `mmcls.datasets`
+
+The documentation can be found [here](mmcls.datasets).
+
+|      Dataset class       |                                    Changes                                     |
+| :----------------------: | :----------------------------------------------------------------------------: |
+|     `CustomDataset`      | Add `data_root` argument as the common prefix of `data_prefix` and `ann_file`. |
+|        `ImageNet`        |                            Same as `CustomDataset`.                            |
+|      `ImageNet21k`       |                            Same as `CustomDataset`.                            |
+|  `CIFAR10` & `CIFAR100`  |               The `test_mode` argument is required argument now.               |
+| `MNIST` & `FashionMNIST` |               The `test_mode` argument is required argument now.               |
```

Comment:
|  `VOC` |        Requires `data_root`  `image_set_path`  and `test_mode` now    |
|  `CUB` |                            Requires `data_root`  and `test_mode` now             |